post-scan-actions/aws-python-teams-notification/handler.py

```diff
@@ -0,0 +1,81 @@
+import urllib3
+import json
+import os
+http = urllib3.PoolManager()
+import textwrap
+
+def lambda_handler(event, context):
+
+    url = os.environ['TEAMS_URL']
+    channel = os.environ['TEAMS_CHANNEL']
+    
+    for record in event['Records']:
+
+        #Message details from SNS event
+        message = json.loads(record['Sns']['Message'])
+        findings = message['scanning_result'].get('Findings')
+
+        #ARN info to get AWS Account ID
+        arn = json.dumps(record['EventSubscriptionArn'])
+        account_id = arn.split(":")[4].strip()
+
+        if findings:
+        
+            malwares = []
+            types = []
+            for finding in message['scanning_result']['Findings']:
+                malwares.append(finding.get('malware'))
+                types.append(finding.get('type'))
+
+            account=str(account_id)
+            
+            malwares=', '.join(malwares)
+            
+            types=', '.join(types)
+            
+            file_url=str(message['file_url'])
+            
+            
+            payload = {
+                       "summary":"Malicious Object Detected",   
+                       "sections":[
+                          {
+                             "activityTitle":"A <b>Malicious object</b> has been detected!"
+                          },
+                          {
+                             "facts":[
+                                {
+                                   "name":"Account ID:",
+                                   "value":account
+                                },
```

Comment:
`facts` use markdown to format texts by default, so markdown formatting characters need to be escaped. As we're using plain text only, suggest to turn it off.
```suggestion
                             "facts":[
                                "markdown": False,
                                {
                                   "name":"Account ID:",
                                   "value":account
                                },
```

Reply:
set markdown for facts section to false.